lang/en/tool_s3logs.php

```diff
@@ -51,6 +51,9 @@
 $string['s3region_desc'] = 'The AWS Region to use for API calls';
 $string['secretkey'] = 'Secret Key';
 $string['secretkey_desc'] = 'The AWS secret key used to make AWS API calls for S3';
-$string['privacy:metadata'] = 's3logs tool export Moodle standard log for archiving purposes';
+$string['privacy:metadata:tool_s3logs:userid'] = 'The userid is sent from Moodle to personalise user activity.';
+$string['privacy:metadata:tool_s3logs:relateduserid'] = 'The userid is sent from Moodle to personalise user activity.';
```

Comment:
Userid?? Isn't this like *everything* not just userid?

Reply:
I was under impression that we need to link just user related data here. So userid and relateduserid are the only fields.